File: php/Merge Sorted Array.php
<?php

class Solution {

  /**
   * @param int[] $nums1
   * @param int $m
   * @param int[] $nums2
   * @param int $n
   * @return null
   */
  function merge(array &$nums1,int $m,array $nums2,int $n):void {
      for ($i = $m; $i < $n + $m; $i++){
          $nums1[$i] = $nums2[$i - $m];
      }
      sort($nums1);
  }
}

File: go/Merge Sorted Array.go
import "slices"

func merge(nums1 []int, m int, nums2 []int, n int) []int  {
    for i := m; i < m + n; i++{
        nums1[i] = nums2[i - m]
    }
    slices.Sort(nums1);
    return nums1
}